event-config.h.cmake

```diff
@@ -473,15 +473,6 @@
 /* The size of 'void *', as computer by sizeof */
 #define EVENT__SIZEOF_VOID_P @EVENT__SIZEOF_VOID_P@
 
-/* set an alias for whatever __func__ __FUNCTION__ is, what sillyness */
-#if defined (__func__)
-#define EVENT____func__ __func__
-#elif defined(__FUNCTION__)
-#define EVENT____func__  __FUNCTION__
-#else
-#define EVENT____func__ __FILE__
-#endif
-
```

Comment:
hm, I'm looking at a46c2609bfa4d68e78665dd55461af9b1ce9651b and it looks to me that `__func__` were introduced with a purpose, have you seen it?

Reply:
AFAICS that patch mentioned why they were introduced (I guess some compiler hadn't implemented `__func__`), but since IRIX is EOL, I think we should clean this up (but not only this two places but also `configure.ac` and `CMakeLists.txt`), and we should check win32 case, since IIRC in some versions of toolchain there is only `__FUNCTION__` and no `__func__`

Reply:
But let's keep it separate from this issue anyway (at least separate patch in this series)

Reply:
See PR #644